src/helm/config/model_deployments.yaml

```diff
@@ -383,6 +404,16 @@ model_deployments:
     window_service_spec:
       class_name: "helm.benchmark.window_services.no_decoding_window_service.NoDecodingWindowService"
 
+  - name: google/text-bison@002
+    model_name: google/text-bison@002
+    tokenizer_name: google/text-bison@002
+    max_sequence_length: 6000 # Officially 8192
+    max_sequence_and_generated_tokens_length: 7000 # Officially 9216
```

Comment:
I think we fixed the tokenizer issue, we can probably revert this and up the official numbers here no?

Reply:
Changed.